README.md

```diff
@@ -163,6 +163,7 @@ If the completion menu does not appear, log in to the remote server and check th
 - `acm-enable-doc-markdown-render`: Richly render Markdown for completion popups, you can choose `'async`, `t` or `nil`. When set to `'async`, styles are applied asynchronously, choose `t`, styles are applied synchronously and will slow down the completion speed, default is `async`
 - `acm-enable-icon`: Whether the complete menu shows the icon, macOS users need to add option `--with-rsvg` to the brew command to install emacs to display SVG icon
 - `acm-enable-tabnine`: Enable tabnine support， enable by default，when enable need execute `lsp-bridge-install-tabnine` command to install TabNine, and it can be used. TabNine will consume huge CPUs, causing your entire computer to be slow. If the computer performance is not good, it is not recommended to enable this option
+- `acm-enable-codeium`: Enable codeium support, when enable need execute `lsp-bridge-install-update-codeium` command to install Codeium, then execute `lsp-bridge-codeium-get-api-key` command to get api key, and it can be used.
```

Comment:
I can't work after execute `lsp-bridge-codeium-get-api-key`, can you show steps how did you got API Key?

Reply:
When the local server is already running, if I execute this command once and log in to the open webpage, the API key will be saved.

You may need to execute `lsp-bridge-codeium-get-api-key` several times. The local server may not have finished starting up on the first execution. I need to think again about how to solve this problem.

Reply:
This situation may also arise because I changed the format of the version and need to rerun lsp-bridge-install-update-codeium to resolve it.